apps/public-docsite-v9/.storybook/main.js

```diff
@@ -21,6 +21,7 @@ module.exports = /** @type {Omit<import('../../../.storybook/main'), 'typescript
     ...getPackageStoriesGlob({ packageName: '@fluentui/react-components', callerPath: __dirname }),
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27289&commitId=62056ba0b78fa7d53e3823ac589b3ab040ce1b0e&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>